test/user_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+
+class UserTest < Minitest::Test
+  def setup
+    @sal = User.new("Sal")
+    @ali = User.new("Ali")
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+    @jokes = [@joke_1, @joke_2]
+  end
+
+  def test_it_exists
+    assert_instance_of User, @sal
+  end
+
+  def test_it_has_a_name
+    assert_equal "Sal", @sal.name
+  end
+
+  def test_it_can_have_another_name
+    assert_equal "Ali", @ali.name
+  end
+
+  def test_it_starts_with_empty_array_of_jokes
+    assert_equal [], @sal.jokes
+  end
```

Comment:
good testing here; you can usually roll all of these "initialized attributes" into a single test. Also we don't really have to test Ali's name; if it works for Sal, it'll work for any User